.NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs

```diff
@@ -96,6 +97,13 @@ protected override void InitializeConfiguration()
                         new GermanMergedParserConfiguration(new BaseOptionsConfiguration(options))),
                     new BaseMergedDateTimeExtractor(new GermanMergedExtractorConfiguration(options))));
 
+            RegisterModel<DateTimeModel>(
```

Comment:
The Italian DateTime model should not be registered as it is not ready. Only the parser/extractor tests should run by default now, until the whole model can be released.